app/src/main/java/pl/pwr/zpi/bcycle/mobile/ui/my_account/MyAccountFragment.kt

```diff
@@ -21,8 +30,39 @@ class MyAccountFragment : Fragment() {
     ): View? {
         myAccountViewModel = ViewModelProvider(this).get(MyAccountViewModel::class.java)
         val root = inflater.inflate(R.layout.fragment_my_account, container, false)
-        val textView: TextView = root.findViewById(R.id.text_my_account)
-        myAccountViewModel.text.observe(viewLifecycleOwner, Observer { textView.text = it })
         return root
     }
+
+    @SuppressLint("CheckResult")
+    override fun onActivityCreated(savedInstanceState: Bundle?) {
+        super.onActivityCreated(savedInstanceState)
+
+        progressBar.visibility = View.VISIBLE
+
+        nameTV.text = auth.currentUser?.displayName;
+        emailTV.text = auth.currentUser?.email;
+        storage.getReferenceFromUrl(auth.currentUser?.photoUrl.toString()).downloadUrl
+            .addOnSuccessListener { Picasso.get().load(it).into(avatarIV) }
+
+
+        ApiClient.statsApi.getUserStats().background().subscribe({
+            stats ->
+            numberOfAllTripsCountTV.text = stats.result.tripCount.toString()
+            numberOfGroupTripsCountTV.text = stats.result.groupTripTotalCount.toString()
+            numberOfCreatedGroupTripsCountTV.text  = stats.result.groupTripHostingCount.toString()
+            totalDistanceKmCountTV.text = stats.result.totalKilometers.toString()
+            totalTimeCountTV.text = stats.result.totalTimeMinutes.toString()
+
+            progressBar.visibility = View.GONE
+        },
+            {error->
+                error.message
```

Comment:
This is an unused variable. Also, the indentation/code style is inconsistent with the success handler (and you don’t need semicolons in Kotlin).